test/distribution/PushDistributor.t.sol

```diff
@@ -0,0 +1,453 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8;
+
+import "forge-std/Test.sol";
+import { IERC20 } from "./../../contracts/tokens/IERC20.sol";
+import { MockERC20 } from "forge-std/mocks/MockERC20.sol";
+import { SetupPartyHelper } from "../utils/SetupPartyHelper.sol";
+import { PushDistributor } from "./../../contracts/distribution/PushDistributor.sol";
+import { ArbitraryCallsProposal } from "./../../contracts/proposals/ArbitraryCallsProposal.sol";
+import { PartyGovernance } from "./../../contracts/party/PartyGovernance.sol";
+import { ProposalExecutionEngine } from "./../../contracts/proposals/ProposalExecutionEngine.sol";
+import { ERC721Receiver } from "./../../contracts/tokens/ERC721Receiver.sol";
+
+contract PushDistributorTest is SetupPartyHelper {
```

Comment:
Don't see a test anywhere that would revert `WrongMembers`. Also not checking revert messages.

Reply:
For some reason, I can't get revert messages to work here with the' ArbitraryCallFailedError` and the `bytes revertData` it returns.

Reply:
@0xble Committed an example of how we can verify revert messages